### PR TITLE
Soback 90 better search field

### PR DESCRIPTION
**Pull request by jesperlauridsen**
Made searchfield better and expandable, began the process of responsive flow and fixed error when searching from field document attributes not removing urlSearch flag.

### CODE REVIEW

File: src/js/src/components/SearchBox.vue [hunk 1]
```diff
@@ -148,6 +151,14 @@ export default {
       updateShowModal:'updateShowModal',
       updateCurrentModal:'updateCurrentModal'
     }),
+    textAreaClick() {
+      console.log('clicks!')
+    },
```
Comment (jorntx) [inline src/js/src/components/SearchBox.vue:156]: ?? :)

File: src/js/src/components/SearchBox.vue [hunk 2]
```diff
@@ -148,6 +151,14 @@ export default {
...
+    getSizeOfTextArea() {
+      let textarea = document.getElementById('query')
+      textarea.style.height = '1px'
+      textarea.style.height = (textarea.scrollHeight) + 'px'
+    },
     selectSearchMethod(selected) {
       console.log(selected)
```
Comment (jorntx) [inline src/js/src/components/SearchBox.vue:163]: Still need this?

**Review by jorntx — COMMENTED** [review]
Looks good! Only minor comments to code and found one small bug --> Size of search box does not resize immediately if reset button is used. Input is resized on first keyUp.

**Review by thomasegense — APPROVED** [review]
OK from me.  The responsive text-areas works perfectly.

File: src/js/src/components/SearchBox.vue (hunk 1 above)
Comment (jesperlauridsen) [reply src/js/src/components/SearchBox.vue:156]: Doh.

File: src/js/src/components/SearchBox.vue (hunk 2 above)
Comment (jesperlauridsen) [reply src/js/src/components/SearchBox.vue:163]: Yep. This is what makes sure the searchbox grows when a new line is added.

Comment (jesperlauridsen) [reply src/js/src/components/SearchBox.vue:163]: removed the console.log tho. If that was what you were refering to :D

Comment (jorntx) [reply src/js/src/components/SearchBox.vue:163]: Yes it was :)

**Review by jorntx — APPROVED** [review]
Good to go!

**Pull request merged**